install-k8s-perf.yml

```diff
@@ -1,3 +1,7 @@
+- hosts: all
+  tasks:
+    - set_fact: prepull_images=['registry.k8s.io/pause:3.1']
```

Comment:
is this `3.1` because the test contains the older one? if yes - wondering if you can submit changes upstream to use latest one.

Reply:
wondering if you can append to the list so that you respect the entries defined in the group_vars file otherwise it doesn't make any sense to have that variable in the group_vars.

Reply:
chatgpt suggests this :)
```
      set_fact:
        prepull_images: "{{ prepull_images + ['registry.k8s.io/pause:3.1'] }}"
```

Reply:
na, just append to the list.. in future if someone needs any other images to be pulled, user can do that via variables file or arg..